hat/assets/js/apps/Iaso/domains/instances/constants.js

```diff
@@ -45,6 +46,21 @@ export const INSTANCE_METAS_FIELDS = [
             );
         },
     },
+    {
+        key: 'planning',
+        type: 'info',
+        renderValue: data => {
+            if (data.planning_id) {
+                return (
+                    <LinkToPlanning
+                        planningId={data.planning_id}
+                        planningName={data.planning_name}
```

Comment:
To be consistent with other Link to, maybe pass a pseudo planning as props:
```
 <LinkToPlanning planning={{
   id: data.planning_id,
  name:  data. planning_name,
}}>
```
This will make the component more generic for further uses